js/modules/loki_app_dot_net_api.js

```diff
@@ -34,6 +48,123 @@ const snodeHttpsAgent = new https.Agent({
 
 const timeoutDelay = ms => new Promise(resolve => setTimeout(resolve, ms));
 
+let adnOnionRequestCounter = 0;
+const sendViaOnion = async (srvPubKey, url, pFetchOptions, options = {}) => {
+  if (!srvPubKey) {
+    log.error(
+      'loki_app_dot_net:::sendViaOnion - called without a server public key'
+    );
+    return {};
+  }
+
+  // set retry count
+  let adnOnionRequestCount;
+  if (options.retry === undefined) {
+    // eslint-disable-next-line no-param-reassign
+    options.retry = 0;
+    adnOnionRequestCounter += 1; // increment counter
+    adnOnionRequestCount = 0 + adnOnionRequestCounter; // copy value
+  } else {
+    adnOnionRequestCount = options.counter;
+  }
+
+  const fetchOptions = pFetchOptions; // make lint happy
```

Comment:
If you only use `fetchOptions` to construct `payloadObj` below, why not use `pFetchOptions` directly (replacing headers with `{}` if needed)?

Reply:
because I only want explicit keys in `payloadObj` and lint wouldn't really like me filtering `pFetchOptions`